documentation/library_guide/macros.rst

```diff
@@ -64,11 +64,34 @@ Macro                              Description
 ---------------------------------- ------------------------------
 ``ONEDPL_USE_TBB_BACKEND``         This macro controls the use of |onetbb_long| or
                                    |tbb_long| for parallel policies.
-                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are only
-                                   executed by the calling thread. This is recommended for code that should not depend on the
-                                   presence of the |onetbb_short| or |tbb_short| library. When the macro is not defined (by default)
-                                   or evaluates to a non-zero value,
+                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are do not
+                                   consider this backend to use and try to find the "next" backend according to the priotiry.
+                                   This is recommended for code that should not depend on the
+                                   presence of the |onetbb_short| or |tbb_short| library.
+                                   When the macro evaluates to a non-zero value, algorithms with the ``par`` and ``par_unseq``
                                    parallel policies are executed using the |onetbb_short| or |tbb_short| library.
+                                   When the macro is not defined (by default) algorithms with the ``par`` and ``par_unseq``
+                                   parallel policies are executed using the |onetbb_short| or |tbb_short| library unless other
+                                   parallel backend is explicitly enabled.
+
+                                   `TBB backend` has higher priority than `OpenMP backend`
+
+                                   Please note that if all parallel backend macros are set to 0 algorithms with
+                                   the ``par`` and ``par_unseq`` policies are executed by the calling thread.
+---------------------------------- ------------------------------
+``ONEDPL_USE_OPENMP_BACKEND``      This macro controls the use of OpenMP* for parallel policies.
+                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are do not
+                                   consider this backend to use and try to find the "next" backend according to the priotiry.
+                                   When the macro evaluates to a non-zero value the algorithms with the ``par`` and ``par_unseq``
+                                   are executed using OpenMP* unless the backend with higher priority is explicitly enabled.
+                                   When this macro is not defined (by default) the algorithms with the ``par`` and ``par_unseq``
+                                   are executed on OpenMP* if dedicated compiler option enables OpenMP (e.g. -fopenmp) and
```

Comment:
Minor: consider -fopenmp -> `-fopenmp`

Reply:
There is no consistency if we render compiler options as a code. I don't mind to do that, though. @akukanov, what's you opinion?

Reply:
I feel we use the code rendering more often than not use it (except maybe for release notes where we prefer to minimize the markdown to keep readability in a plain text mode). In the pages being changed by this patch, I only found ``-fsycl-unnamed-lambda``, used in backquotes. I'd follow the same style.

Reply:
we have -O0 rendered as non-code style. I could change that as well

Reply:
done